cmd/kubeadm/app/cmd/token.go

```diff
@@ -255,8 +263,19 @@ func RunGenerateToken(out io.Writer) error {
 	return nil
 }
 
+type outputItem struct {
```

Comment:
let's name this something more specific:
`outputItemToken`

Reply:
Done